src/php_spx.c

```diff
@@ -362,6 +362,12 @@ static int check_access(void)
     }
 
     const char * authorized_ips_str = SPX_G(http_ip_whitelist);
+
+    if (0 == strcmp(authorized_ips_str, "*")) {
```

Comment:
You must move this `if` right after the next one (i.e. https://github.com/NoiseByNorthwest/php-spx/blob/daf396727cb3e808a7bea909d2fc5b8d7c3137c7/src/php_spx.c#L385), it will be clearer and the null-pointer case will be checked before.

Reply:
Thanks for the feedback, I changed it.